pi5/board.go

```diff
@@ -274,26 +353,30 @@ func (b *pinctrlpi5) AnalogNames() []string {
 }
 
 // DigitalInterruptNames returns the names of all known digital interrupts.
+// Unimplemented because we do not have an api to communicate this over.
 func (b *pinctrlpi5) DigitalInterruptNames() []string {
-	if b.interrupts == nil {
-		return nil
-	}
-
-	names := []string{}
-	for name := range b.interrupts {
-		names = append(names, name)
-	}
```

Comment:
i changed how interrupts were being stored to use the bcom, but also this function doesn't actually do anything in modules anyway so I removed it